buildSrc/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts

```
@@ -59,6 +64,9 @@ extraJavaModuleInfo {
     automaticModule("io.prometheus:simpleclient_httpserver", "io.prometheus.simpleclient.httpserver")
 
     automaticModule("j2objc-annotations-1.3.jar", "j2objc.annotations")
+    automaticModule("io.perfmark:perfmark-api", "perfmark.api")
```

`io.perfmark:perfmark-api:0.26.0` defines `io.perfmark` as automatic module name. We should use the same.
```suggestion
    automaticModule("io.perfmark:perfmark-api", "io.perfmark")
```